### PR TITLE
Team page updates

### PR DESCRIPTION
**Pull request by ajinkyaa**
Team page updates
Screenshot 2022-06-15 at 4 23 38 PM
Screenshot 2022-06-15 at 4 23 23 PM

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@ajinkyaa Please check the CI tests that are failing for this PR.

**Comment by ajinkyaa** [comment]
@supriya3105 I think someone from ror side to work on this. Those are failing because new routes addition for teams page.

File: app/javascript/src/components/Team/List/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useList } from "context/TeamContext";
+import { useUserContext } from "context/UserContext";
+import { MagnifyingGlass, Plus } from "phosphor-react";
+
+const Header = () => {
+  const { isAdminUser } = useUserContext();
+  const { setModalState } = useList();
+  console.log("isAdminUser ---> ", isAdminUser);
```
Comment (JijoBose) [inline app/javascript/src/components/Team/List/Header.tsx:9]: Let's remove console logs

File: app/javascript/src/components/Team/modals/Modals.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const Modals = () => {
+  const { modal } = useList();
+
+  switch (modal) {
+    case "addEdit":
+      return <AddEditMember />;
+    case "delete":
+      return <DeleteMember />;
+    default:
+      return <></>;
```
Comment (JijoBose) [inline app/javascript/src/components/Team/modals/Modals.tsx:16]: Can we add `null` here?

**Review by JijoBose — APPROVED** [review]

**Review by JijoBose — CHANGES_REQUESTED** [review]
tests are failing

**Review by JijoBose — APPROVED** [review]

**Pull request merged**